scanner/src/main/kotlin/ScanContext.kt

```diff
@@ -53,5 +54,11 @@ data class ScanContext(
      * be used by scanners where scan results are stored, because then changes in the mapping would not be applied to
      * stored results.
      */
-    val detectedLicenseMapping: Map<String, String> = emptyMap()
+    val detectedLicenseMapping: Map<String, String> = emptyMap(),
+
+    /**
+     * The packages known to be covered in the context of this scan. For package scanners, this is the list of packages
+     * that have the same provenance as the reference package.
+     */
+    val coveredPackages: List<Package> = emptyList()
```

Comment:
@mnonnenmacher Are package scanners actually more like a deprecated / phased out thing, or is there plans to keep them? (I wonder if it would make sense to introduce a dedicated `PackageScanContext` class, to not pollute the provenance scan user case with these attributes.)

Reply:
There are currently no plans to remove `PackageScannerWrapper`, at least as long as `FossId` is not migrated.